tests/migration_test.go

```diff
@@ -820,7 +811,7 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 				By("Waiting for NFS pod to disappear")
 				tests.WaitForPodToDisappearWithTimeout(tests.NFSTargetName, 120)
 			})
-			It("[test_id:2653]  should be migrated successfully, using guest agent on VM", func() {
+			PIt("[test_id:2653]  should be migrated successfully, using guest agent on VM", func() {
```

Comment:
will need to adjust to a real skip

Reply:
can i usetests.SkipPVCTestIfRunnigOnKindInfra() ?
since it runs NewRandomVMIWithPVC

Reply:
yes, but we need to make sure Adam's PR is indeed fixing it. If not, we should keep investigating.

Reply:
adam PR is still WIP, and is needed for it , so we better wait with it and focus on the other first ?

Reply:
Adam's Pr is ready, just has some ci issues.
You can meanwhile take the commit from Adam's PR and add it to your PR to test if it fixes this test.

If fixes, you can use the tests.SkipPVCTestIfRunnigOnKindInfra(). Otherwise, you will need to introduce a new skip.

Reply:
Storage  Starting a VirtualMachineInstance  [rfe_id:2298][crit:medium][vendor:cnv-qe@redhat.com][level:component]  With HostDisk and PVC initialization With smaller than requested PVCs  [test_id:3109]Should initialize an empty PVC with a disk.img when disk  is too small but within toleration expand_less | 38s
-- | --
/root/go/src/kubevirt.io/kubevirt/tests/storage_test.go:577 Unexpected Warning event received: testvmisj68cp8p66cf9cm2vdzz5wq585hd72lxwprqz6w4xlmq2cfn,1db27be8-9653-41bc-b23c-d06fad462249: server error. command SyncVMI failed: "preparing host-disks failed: unable to create /tmp/kubevirt/wzz5b89nh7/mount/disk.img, not enough space, demanded size 227781854400 B is bigger than available space 134950559744 B, also after taking 30 % toleration into account"


this test started to fail , maybe lets wait for the PR of Adam to be merged and this storage issues to be fixed and then we can try at once?

Reply:
as we spoke, it doesnt fix it yet, we will skip and back to it in following PR once its merged